test/public-stats.test.js

```diff
@@ -4,9 +4,10 @@ import pg from 'pg'
 import { DATABASE_URL } from '../lib/config.js'
 import { migrateWithPgClient } from '../lib/migrate.js'
 import { buildEvaluatedCommitteesFromMeasurements, VALID_MEASUREMENT } from './helpers/test-data.js'
-import { updatePublicStats } from '../lib/public-stats.js'
+import { updateDailyClientRetrievalStats, updatePublicStats } from '../lib/public-stats.js'
 import { beforeEach } from 'mocha'
 import { groupMeasurementsToCommittees } from '../lib/committee.js'
+import { describe } from 'node:test'
```

Comment:
We are using `mocha` in this repository. Mocha defines `describe` and `it` as global variables.

Please remove this line.

```suggestion
```